### PR TITLE
refactor: hide non-functional elements in highlights card

### PR DESCRIPTION
**Pull request by OgDev-01**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR hides all elements that need API updates to be functional for now

### Before
image

image

### After
image

image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
https://github.com/open-sauced/insights/issues/887
https://github.com/open-sauced/insights/issues/886
https://github.com/open-sauced/insights/issues/885
https://github.com/open-sauced/insights/issues/884
https://github.com/open-sauced/insights/issues/883
https://github.com/open-sauced/insights/issues/882

### CODE REVIEW

**Review by bdougie — APPROVED** [review]

**Comment by bdougie** [comment]
@OgDev-01, I added a list of issues to the 'what needs to be done after this is deployed" section.

File: components/molecules/ContributorHighlight/contributor-highlight-card.tsx [hunk 1]
```diff
@@ -96,13 +96,13 @@ const ContributorHighlightCard = ({ title, desc, prLink, user }: ContributorHigh
                   <span>Copy link</span>
                 </div>
               </DropdownMenuItem>
-              <DropdownMenuItem className="rounded-md">
+              <DropdownMenuItem className="rounded-md hidden">
```
Comment (0-vortex) [inline components/molecules/ContributorHighlight/contributor-highlight-card.tsx:99]: Screenshot 2023-02-20 at 22 17 22

Comment (0-vortex) [reply components/molecules/ContributorHighlight/contributor-highlight-card.tsx:99]: something like this but for users to users?

Comment (bdougie) [reply components/molecules/ContributorHighlight/contributor-highlight-card.tsx:99]: Not clear. Are you suggesting a follower endpoint similar to the stargaze one?

Comment (bdougie) [reply components/molecules/ContributorHighlight/contributor-highlight-card.tsx:99]: nvm, this is for a reaction service. Looking at the code.

Comment (0-vortex) [reply components/molecules/ContributorHighlight/contributor-highlight-card.tsx:99]: no just trying to understand what "api updates" we are waiting on. these requirements should be mentioned in the design process, not when we are reviewing features to go into live testing environment :/

Comment (bdougie) [reply components/molecules/ContributorHighlight/contributor-highlight-card.tsx:99]: I agree.

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**